### PR TITLE
Implement crate dropping delay and power up activation delay

### PR DESCRIPTION
**Pull request by agustinesco**
>[!Warning]
> This pr should be merged alongside [This backend branch](https://github.com/lambdaclass/mirra_backend/pull/735)

## Motivation

We need some time before spawning the power ups when the crates break to play an animation

### CODE REVIEW

File: client/Assets/Scripts/PowerUpsManager.cs [hunk 1]
```diff
@@ -33,7 +33,7 @@ public void UpdatePowerUps()
             PowerUp powerUp = powerupEntity.PowerUp;
 
             if (
-                powerUp.Status == PowerUpstatus.Available
+                powerUp.Status == PowerUpstatus.Unavailable
```
Comment (tvillegas98) [inline client/Assets/Scripts/PowerUpsManager.cs:36]: I understand that if we keep the former check, the power up will not spawn until the `activation_time` has passed right?

**Review by tvillegas98 — APPROVED** [review]

File: client/Assets/Scripts/PowerUpsManager.cs (hunk 1 above)
Comment (agustinesco) [reply client/Assets/Scripts/PowerUpsManager.cs:36]: exactly, this change was made because i encountered that error

**Review by tkz00 — APPROVED** [review]
Great work Agus, I feel a longer delay to pick up the power ups could feel better, but that is just configuration work, these changes look and work great! Tested on my android.

**Review by manucamejo — APPROVED** [review]

**Review by tvillegas98 — APPROVED** [review]
Great job!

**Pull request merged**